frontend/src/i18n/en.js

```diff
@@ -509,11 +509,29 @@ const en = {
 
   widget: {
     cubejs: {
+      tooltip: {
+        Activities: {
+          singular: 'Activity',
```

Comment:
Instead of having to add the singular/plural logic you can just set the singular format, for example: `Activities: 'Activity'`, and then when getting the translation you could use `pluralize` (we are already using in some places, automation-list-page.vue):
```
callbacks: {
          label: (context) => {
            return pluralize(
              i18n(`widget.cubejs.tooltip.${context.dataset.label}`),
              context.dataset.data[context.dataIndex],
              true
            )
          }
        }
```
This will already do all the logic to render it as:
1 Member
2 Members

Reply:
Also updated - we're using pluralize now and only keeping the singular form in translations